doc/source/whatsnew/v1.1.0.rst

```diff
@@ -1181,23 +1180,23 @@ ExtensionArray
 ^^^^^^^^^^^^^^
 
 - Fixed bug where :meth:`Series.value_counts` would raise on empty input of ``Int64`` dtype (:issue:`33317`)
-- Fixed bug in :func:`concat` when concatenating DataFrames with non-overlaping columns resulting in object-dtype columns rather than preserving the extension dtype (:issue:`27692`, :issue:`33027`)
+- Fixed bug in :func:`concat` when concatenating :class:`DataFrame`s with non-overlapping columns resulting in object-dtype columns rather than preserving the extension dtype (:issue:`27692`, :issue:`33027`)
```

Comment:
unwanted pattern ci failure - backticks incorrectly rendering because of missing spaces

Reply:
Thanks - passing now.